app/controllers/members_controller.rb

```diff
@@ -1,3 +1,8 @@
+# This controller allowed community users to manage their organisation's members.
+# They could invite new members and remove members from the organisation.
+#
+# This functionality was removed in Dec 2020 (commit 6bc2ea7a) but the code
+# has been left in, presumably in case we want to add it back in.
```

Comment:
This is included because I was confused about when the `invite_from_community_email` gets sent... turns out, the answer is never!